async.go

```diff
@@ -193,3 +195,73 @@ func (sr *snowflakeRestful) getAsyncOrStatus(
 
 	return response, nil
 }
+
+// panic message for no response from get func
+type panicMessageType = struct {
+	deadlineSet bool
+	deadline    time.Time
+	startTime   time.Time
+	timeout     time.Duration
+	statusCode  string
+	stack       *[]uintptr
+}
+
+func getPanicMessage(
```

Comment:
this could be called `new...`. It should also be defined in another file (makes later rebasing if our fork easier)

Reply:
ok sounds good